test/smoke/package/src/debugMacOS.test.ts

```diff
@@ -131,9 +131,7 @@ export function startDebugMacOSTests(
                 `${testname}: Searching for "Test output from debuggee" string in console`,
             );
             await automationHelper.runCommandWithRetry("Debug: Focus on Debug Console View");
-            let found = await app.workbench.debug.waitForOutput(output =>
-                output.some(line => line.indexOf("Test output from debuggee") >= 0),
-            );
+            let found = await automationHelper.waitForOutputWithRetry("Test output from debuggee");
```

Comment:
Could you also change it with `const`?

```suggestion
            const found = await automationHelper.waitForOutputWithRetry("Test output from debuggee");
```